.github/workflows/test-and-release.yml

```diff
@@ -18,14 +20,14 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [18.x, 20.x]
```

Comment:
Looks good! All Projects are 18+ compatible?

Reply:
It's only the tests for renovate that are run using 18 and 20. Does not relate to projects.